common/src/registers/MachRegister.C

```diff
@@ -769,6 +790,9 @@ namespace Dyninst {
           return true;
       }
     }
+    if(arch == Arch_riscv64) {
```

Comment:
We don't need this check since `getZeroFlag` will return `InvalidReg`. This puts the documentation for riscv in one spot (i.e., in `getZeroFlag`).

Reply:
Fixed